aspnetcore/signalr/groups.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Manage users and groups in SignalR
+author: rachelappel
+description: Overview of ASP.NET Core SignalR User and Group management.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/04/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/groups
+---
+
+# Manage users and groups in SignalR
+
+By [Brennan Conroy](https://github.com/BrennanConroy)
+
+Users and groups is how SignalR exposes connection management.
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/signalr/groups/sample/) [(how to download)](xref:tutorials/index#how-to-download-a-sample)
+
+## What are users?
```

Comment:
nit: Maybe make this "SignalR Users", "Users in SignalR" etc. Just a though.
Just sounds odd to me as "What are users"

Reply:
What even are users?

I like "Users in SignalR"